src/utils/cplay.c

```diff
@@ -564,6 +671,33 @@ void get_codec_iec(FILE *file, struct compr_config *config,
 	codec->format = 0;
 }
 
+static int check_stdin(struct compress *compress)
+{
+	switch(do_pause()) {
+		case DO_PAUSE:
+			if (compress_pause(compress) != 0) {
+				fprintf(stderr, "Pause ERROR\n");
+				return -1;
+			}
+			is_paused = true;
+			break;
+		case DO_RESUME:
+			if (compress_resume(compress) != 0) {
+				fprintf(stderr, "Resume ERROR\n");
+				return -1;
+			}
+			is_paused = false;
+			break;
+		case DO_NOTHING:
+			break;
+		default:
+			fprintf(stderr, "do_pause unexpected case.\n");
```

Comment:
print the value of do_pause here, helps in understand why this error was flagged

Reply:
Removed the default case altogether as `do_pause()` can only return `DO_NOTHING,` `DO_PAUSE` or `DO_RESUME` and nothing else so it was kinda pointless.